packages/ses/docs/lockdown.md

```diff
@@ -366,12 +367,54 @@ the container to exit explicitly, and we highly recommend setting
 - `'none'`: do not install traps for uncaught exceptions. Errors are likely to
   appear as `{}` when they are reported by the default trap.
 
+## `evalTaming` Options
+
+This option only affects the initial compartment!
```

Comment:
I've been using "start compartment" rather than "initial compartment", but I defer to @kriskowal on this.

```suggestion
This option only affects the start compartment!
```